tests/python/pants_test/pants_run_integration_test.py

```diff
@@ -612,15 +620,27 @@ def do_command(self, *args, **kwargs):
     """Wrapper around run_pants method.
 
     :param args: command line arguments used to run pants
-    :param kwargs: handles 1 key
-      success - indicate whether to expect pants run to succeed or fail.
     :return: a PantsResult object
     """
-    success = kwargs.pop('success', True)
     cmd = list(args)
+    success = kwargs.pop('success', True)
```

Comment:
This could be a python 3 TODO to make `success` a keyword-only argument (see #7076).